demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1495,14 +1528,12 @@ def test_modeling_rule(
             # can ignore the types since if they are not set to str values an error occurs
             xsiam_url,  # type: ignore[arg-type]
             retrying_caller,
-            api_key,  # type: ignore[arg-type]
-            auth_id,  # type: ignore[arg-type]
-            xsiam_token,  # type: ignore[arg-type]
-            collector_token,  # type: ignore[arg-type]
             push,
             interactive,
             ctx,
             delete_existing_dataset,
+            xsiam_client=xsiam_client,
+            tenant_demisto_version=Version(tenant_demisto_version),
```

Comment:
why not to return it as a Version object from the xsiam_client?

Reply:
True, I'll update it.